packages/augur-sdk/src/state/db/LiquidityDB.ts

```diff
@@ -95,11 +104,16 @@ export class LiquidityDB extends AbstractDB {
     }
   }
 
+  /**
+   * Deletes hourly liquidity data older than 24 hours
+   * @param liquidityDB
+   * @param mostRecentOnTheHourTimestamp
+   */
   private async deleteOldLiquidityData(liquidityDB: LiquidityDB, mostRecentOnTheHourTimestamp: BigNumber): Promise<void> {
     const oldLiquidityDocs = await liquidityDB.find({
       selector: {
         _id: { $ne: 'lastUpdated' },
-        timestamp: { $lte: mostRecentOnTheHourTimestamp.minus(SECONDS_IN_A_DAY).toNumber() },
+        timestamp: { $lt: mostRecentOnTheHourTimestamp.minus(SECONDS_IN_A_DAY).toNumber() },
```

Comment:
it seems like liquidity db can be deleted as soon as a new one is created. I don't think we need to keep 1 whole days liquidity dbs around.

Reply:
The liquidity db stores the hourly liquidity for each spread for a market for the last 24 hours. After 24 hrs, we can delete any old hourly data, which is what this is querying for.

Reply:
We do need to keep 24 hours worth of data in order to implement the liquidity depleted filter, according to what Joey had said on Discord.

Reply:
ok, I was under the impression that the market had an 1 hour to fix their spread. and the `recently depleted` would allow users to find their market for 1 hour. 

Giving them 24 hours makes sense as well. So if they don't fix their spread with in 24 hours does their market no longer show up if `recently depleted` is selected?

Reply:
Correct, according to Joey, a market's liquidity is considered recently depleted if it had liquidity under a 15% spread in the last 24 hours, but doesn't currently have liquidity.